api/env_test.go

```diff
@@ -0,0 +1,44 @@
+package api
+
+import (
+	"os"
+	"testing"
+)
+
+func TestReadBaoVariable_Vault(t *testing.T) {
+	actual := "example_value"
+	os.Setenv("VAULT_TEST", actual)
+	expected := ReadBaoVariable("BAO_TEST")
+	if actual != expected {
```

Comment:
For the future, the project is really inconsistent about using it, but there's a library [called `require`](https://pkg.go.dev/github.com/stretchr/testify/require) that lets you do one-line assertions common in other languages. Three lines here isn't bad, so I'm happy without it but if you find yourself with more unit tests, don't feel you need to write a bunch of if statements. :-)